src/test/java/com/redhat/devtools/lsp4ij/features/selectionRange/TypeScriptSelectionRangeTest.java

```diff
@@ -18,15 +18,18 @@
  */
 public class TypeScriptSelectionRangeTest extends LSPSelectionRangeFixtureTestCase {
 
+    private static final String DEMO_TS_FILE_NAME = "demo.ts";
```

Comment:
No new tests were added here, but I extracted common constants for repeated strings for consistency with what I did in the new tests added in this PR.